solution/backend/regulations/templates/regulations/statute.html

```diff
@@ -18,6 +18,7 @@
 {% endblock %}
 
 {% block body %}
+{{ site_config|json_script:"site_config" }}
```

Comment:
Passing in complex values using the Django best practice discussed here: https://adamj.eu/tech/2022/10/06/how-to-safely-pass-data-to-javascript-in-a-django-template/